km/km_filesys.c

```diff
@@ -296,27 +391,30 @@ int km_shrink_footprint(km_vcpu_t* vcpu)
 {
    char* envarray[5];
    char* argv[3];
-   char listeningfd[32];
    char timeout[32];
    char kmverbose[32];
    char me[128];
    char* tmp;
 
+   /*
+    * If we aren't running a snapshot or the SNAP_LISTEN_TIMEOUT
+    * env var is undefined, then km will not stay in shrunken state
+    * after execing back to km, so there is no reason to attempt
```

Comment:
Guess I'm being dyslexic. You want me to change the word to itself?
Are you asking me to put a quote in there but github took it out because it knows better?

Reply:
Now looking at it, I have no idea what I meant. Somehow it looked misspelled to me. I guess I am dyslexic ...